### DISCUSSION TITLE
Optional TempFile field

### DISCUSSION THREAD

**Discussion by XenorInspire** [rwf2/Rocket#2835] [Questions] [answered]
Hi,
First, thanks for all the work you are doing for this project !
My question is : is it possible to have a default value for a TempFile field ?
I have a struct which derives FromForm, and in my html form sometimes the user can upload a file and sometimes not (optional field).
But I didn't find how to tell to the rocket server that TempFile is an optional field ?

Thanks again,
Best Regards,
Xen0r.

**Comment by the10thWiz** [comment, marked as answer]
You can make any field optional using `Option<T>`.

**Reply by XenorInspire** [reply]
Oh perfect !
Thanks a lot x)
